libs/klv/src/test/java/org/codice/alliance/libs/klv/TestGeoBoxHandler.java

```diff
@@ -140,4 +146,53 @@ public void testAcceptData() throws KlvDecodingException {
                 is("POLYGON ((2.000000 1.000000, 4.000000 3.000000, 6.000000 5.000000, 8.000000 7.000000, 2.000000 1.000000))"));
 
     }
+
+    @Test
+    public void testTrim() throws KlvDecodingException {
+
+        geoBoxHandler.accept(KlvUtilities.createTestFloat(LAT1, 1));
+        geoBoxHandler.accept(KlvUtilities.createTestFloat(LON1, 2));
+        geoBoxHandler.accept(KlvUtilities.createTestFloat(LAT2, 3));
+        geoBoxHandler.accept(KlvUtilities.createTestFloat(LON2, 4));
+        geoBoxHandler.accept(KlvUtilities.createTestFloat(LAT3, 5));
+        geoBoxHandler.accept(KlvUtilities.createTestFloat(LON3, 6));
+        geoBoxHandler.accept(KlvUtilities.createTestFloat(LAT4, 7));
+        geoBoxHandler.accept(KlvUtilities.createTestFloat(LON4, 8));
+
+        // these are the values that should get trimmed
+        geoBoxHandler.accept(KlvUtilities.createTestFloat(LAT1, 1));
```

Comment:
If I'm understanding this correctly, this test could be made stronger by using different lat/lon values from the ones you used above in these 4 calls. Since these values are the same as the ones you used above, the assertions below can't prove which value was actually trimmed.